bionlp/taggers/rnn_feature/tagger_utils.py

```diff
@@ -123,6 +123,41 @@ def trim_tags(tagged_data):
                           for x, y in tagged_data[i]]
     return tagged_data
 
+def get_regular_or_capitalized_embeddings(w2i, i2w, mdl, params):
```

Comment:
you need w2i only in line 129 to define the iterator. But isn't `len(w2i)==len(i2w)`? If so, you can remove it from the signature